editor.go

```diff
@@ -17,53 +17,77 @@ func NewEditor() *Editor {
 
 //go:generate gentut -t Editor -p owasp -in editor.go -w
 type Editor struct {
-	entries []Entry
+	Entries []Entry
+}
+
+// SetApplicableByLevel sets the all entries with the given level or lower as applicable
```

Comment:
Thank you for your MR!

Here, document the appropriate values for levels so users know what is ok as input.

Reply:
or use the below example of using a `type Level int`